starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -799,4 +800,24 @@ protected boolean getIsAmatiDevice() {
   protected String getBuildFingerprint() {
     return Build.FINGERPRINT;
   }
+
+  @SuppressWarnings("unused")
+  @UsedByNative
+  protected long getPlayServicesVersion() {
+    try {
+      if (android.os.Build.VERSION.SDK_INT < 28) {
+        return appContext
+            .getPackageManager()
+            .getPackageInfo(GOOGLE_PLAY_SERVICES_PACKAGE, 0)
```

Comment:
getPackageInfo is deprecated in API level 33. We should do a if else branch here.